src/renderer/components/TaskProgress.ts

```diff
@@ -0,0 +1,85 @@
+import { Task, TaskRuntime, TaskHandle } from "@xmcl/task";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['TaskHandle' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5247069)